pcsx2/DEV9/pcap_io.cpp

```diff
@@ -370,6 +379,12 @@ PCAPAdapter::PCAPAdapter()
 	host_mac = hostMAC;
 	ps2_mac = newMAC; //Needed outside of this class
 
+	if (pcap_io_init(config.Eth, config.EthApi == NetApi::PCAP_Switched, newMAC) == -1)
+	{
+		Console.Error("Can't open Device '%s'\n", config.Eth);
```

Comment:
DEV9: prefix would be nice for the log, also remove the \n in the end.

Reply:
Done